mlir/unittests/programs/qco_programs.cpp

```
@@ -3379,6 +3452,41 @@ SmallVector<Value> nestedForLoopWhileOp(QCOProgramBuilder& b) {
   return measureAndReturnQTensor(b, scfFor[0], 2);
 }
 
+SmallVector<Value> nestedForLoopSwitchOp(QCOProgramBuilder& b) {
```

Cppcheck (2.21.0)

[style] The function 'nestedForLoopSwitchOp' is never used.

(unusedFunction)